tox/config.py

```diff
@@ -818,7 +819,7 @@ def make_envconfig(self, name, section, subs, config):
             atype = env_attr.type
             if atype in ("bool", "path", "string", "dict", "dict_setenv", "argv", "argvlist"):
                 meth = getattr(reader, "get" + atype)
-                res = meth(env_attr.name, env_attr.default)
+                res = meth(env_attr.name, env_attr.default, replace=replace)
```

Comment:
**Note to reviewers**: adding this here is why I needed to add a `replace` argument all over the place.